src/common/components/ui/ClusterEventsToolbar.css

```diff
@@ -1,3 +1,12 @@
 #checkbox-deselect-all-hosts-action {
   visibility: hidden;
 }
+
+#clusters-events-toolbar {
```

Comment:
I realize we do not follow BEM closely in this lib but we should.
Please change the selector to class name.

Also once you're at it, can you change the id selector above to class name selector too ? This is the only file with id selectors so it would be nice to get rid of them all :)

Reply:
It appears the first one is never used. Should I remove it?